Tiltfile

```diff
@@ -123,3 +129,43 @@ local_resource(
     serve_cmd=searcher_command,
     resource_deps=["deploy-contracts", "auction-server", "create-configs"],
 )
+
+
+# Solana resources
```

Comment:
let's add svm prefix to these resources to easily distinguish them on the tilt ui
same for evm